.github/workflows/pytket_benchmarking.yml

```diff
@@ -77,6 +76,7 @@ jobs:
         run: |
           pip install pytket-*.whl
           pip install --pre --index-url https://github_actions:${{ secrets.PRIVATE_PYPI_PASS }}@cqcpythonrepository.azurewebsites.net/simple/ pytket_benchmarking
+          pip list
```

Comment:
Is this needed?

Reply:
I would like just to see which version of pytket is being installed the first couple of times this runs. Just wanting to be careful that installing pytket_benchmarking does not require a conflicting version. I think I have checked this before but would like to be doubly sure.